apps/prairielearn/src/pages/enroll/enroll.ts

```diff
@@ -100,11 +103,15 @@ router.post('/', [
       flash('success', `You have joined ${courseDisplayName}.`);
       res.redirect(req.originalUrl);
     } else if (req.body.__action === 'unenroll') {
-      await queryZeroOrOneRowAsync(sql.unenroll, {
-        course_instance_id: req.body.course_instance_id,
-        user_id: res.locals.authn_user.user_id,
-        req_date: res.locals.req_date,
-      });
+      await queryOptionalRow(
```

Comment:
Again, `queryAsync`?

Reply:
Done in 896f595a8300398a754c7427781bb51eba87d9c6.